builders/tools/hdf5.py

```diff
@@ -14,10 +14,13 @@ def install(dir_name,version=None):
         try:
             tmp_dir = tempfile.mkdtemp()
             path = os.path.join(tmp_dir,name)
-            url = os.path.join('http://www.hdfgroup.org/ftp/HDF5/releases/hdf5-'+version,'src',name)
+            # url = os.path.join('http://www.hdfgroup.org/ftp/HDF5/releases/hdf5-'+version,'src',name)
+            url = os.path.join("https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-" + version, 'src', name)

```

Comment:
Just remove the old url instead of commenting it out, since it clearly doesn't work anymore. Looks like they redid their entire website.